### PR TITLE
Delete zvent.dm

### PR DESCRIPTION
**Pull request by MrsTonedOne**
We don't use it, its ass, it likely doesn't even work, and space whats his name will be making something better.

### CODE REVIEW

File: code/modules/atmospherics/machinery/other/zvent.dm [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-	//all this object does, is make its turf share air with the ones above and below it, if they have a vent too.
-	if(isturf(loc)) //if we're not on a valid turf, forget it
-		for (var/new_z in list(-1,1))  //change this list if a fancier system of z-levels gets implemented
-			var/turf/open/zturf_conn = locate(x,y,z+new_z)
-			if (istype(zturf_conn))
-				var/obj/machinery/zvent/zvent_conn= locate(/obj/machinery/zvent) in zturf_conn
-				if (istype(zvent_conn))
-					//both floors have simulated turfs, share()
-					var/turf/open/myturf = loc
-					var/datum/gas_mixture/conn_air = zturf_conn.air //TODO: pop culture reference
```
Comment (duncathan) [inline code/modules/atmospherics/machinery/other/zvent.dm]: rest in peace this comment

**Review by duncathan — APPROVED** [review]
fwiw it does actually work it's just bad

File: code/modules/atmospherics/machinery/other/zvent.dm (hunk 1 above)
Comment (Cyberboss) [reply code/modules/atmospherics/machinery/other/zvent.dm]: F

**Comment by lzimann** [comment]
#30226

**Comment by Cyberboss** [comment]
Also #35339

**Comment by MrStonedOne** [comment]
@KorPhaeron
They are mainly being removed because its a broken concept. it links two vents on different z-levels such that they exchange air but it does this in a way that doesn't take advantage of process cell, doesn't unite the atmos changes under an excited group so we can't avg out all effected turfs when the difference gets low, it treats the two turfs as if they are ajoining pipenets taking advantage of none of our turf based code or optimizations

**Comment by duncathan** [comment]
@KorPhaeron if you would like this functionality for a multiz map let me know and I will happily make pipes properly multiz. this is a simpler solution for a simpler time.

**Pull request merged**